runtime/ts/particle.ts

```diff
@@ -70,7 +88,7 @@ export class Particle {
    *    added: An Array of Entities added to a Collection-backed Handle.
    *    removed: An Array of Entities removed from a Collection-backed Handle.
    */
-  onHandleUpdate(handle, update) {
+  onHandleUpdate(handle: Handle, update) {
```

Comment:
I think update is {added: Entity[], removed: Entity[]}

Reply:
There's also data and originator.  the call paths are a little hard to trace atm so I put them as any.